jira.js

```diff
@@ -1,22 +1,23 @@
 //Calls what is needed for JIRA
-function jiraSetup() {
+function jiraSetup(url, image) {
   var xhr = new XMLHttpRequest();
-  createIssue(xhr);
+  createIssue(xhr, url, image);
 }
 
 //Creates an Issue in JIRA
-function createIssue(xhr){
+function createIssue(xhr, url, image){
   var projID = 11300 //CONSULTING ID
```

Comment:
should (some of) these be dynamic? assuming this is just a todo?

Reply:
Yeah, I believe projID will always be that value. IssueID changes if it is a bug item (in the following pr) and still need to implement account.